numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -58,6 +59,22 @@
     NvJitLinker, NvJitLinkError = None, None
 
 
+USE_NV_BINDING = config.CUDA_USE_NVIDIA_BINDING
+
+if USE_NV_BINDING:
+    from cuda.bindings import driver as binding
+    from cuda.core.experimental import (
+        Linker,
+        LinkerOptions,
+        ObjectCode,
+    )
+
+    # There is no definition of the default stream in the Nvidia bindings (nor
+    # is there at the C/C++ level), so we define it here so we don't need to
+    # use a magic number 0 in places where we want the default stream.
+    CU_STREAM_DEFAULT = 0
```

Comment:
This isn't true. There is a default stream defined in the CUDA Python bindings, mirroring the same default value in the C/C++ runtime.

```python
>>> from cuda.bindings import runtime
>>> runtime.cudaStreamDefault
0
```

Also if this is kept, please write NVIDIA in all caps.
```suggestion
    # There is no definition of the default stream in the NVIDIA bindings (nor
    # is there at the C/C++ level), so we define it here so we don't need to
    # use a magic number 0 in places where we want the default stream.
    CU_STREAM_DEFAULT = 0
```

Reply:
I'm not sure how important the distinction is, but that's from the runtime binding rather than the driver binding. Is there one defined in the driver binding?

(for background - that statement was true at one point, but may have become false at some point if a definition were added to the NVIDIA bindings)